docs/quickstart/cli.md

```diff
@@ -0,0 +1,112 @@
+---
+title: Using the Redoc CLI
+---
+
+# Using the Redoc CLI
+
+With Redoc's command-line interface you can locally render your
+OpenAPI definition with Redoc and bundle your OpenAPI definition and Redoc into
```

Comment:
"and bundle your OpenAPI definition **and Redoc**"

I'm slightly confused about this "and Redoc" part - does that mean "your OpenAPI definition and API documentation (made with Redoc)"?

Reply:
will reword